lib/pod_ident.rb

```diff
@@ -38,7 +36,7 @@ def find_rule
         match = rule.fetch(:match)
 
         found = apply_starts_with(match['startsWith'], found)
-        found = apply_includes(match['includes'], found)
+        found ||= apply_includes(match['includes'], found)
```

Comment:
If there is a `startsWith` and an `includes` rule in one app, the second one does not need to run if the first one matches.

Reply:
👍